### ISSUE TITLE
[joss paper] remarks

### ISSUE THREAD

**Issue by lrnv** [EnergyModelsX/EnergyModelsBase.jl#27]
Hi ! A few notes on the paper while i read it : 

- The example section is abit weird, as, for me (I am not an expert in your field), i did not understand at all what your software did optimize based on what data to obtain the proposed graph. Could you enlarge a bit the section to detail clearly what are 1) Inputs, 2) aims, goals, algorithms, etc.. 3) Outputs ? 
- While the zenodo archive and the repositories are all on MIT licence, the paper shows out CC By 4.0. This is not necessarily an issue but you might want to sync up (or not). 

xref https://github.com/openjournals/joss-reviews/issues/6619

**Comment by hellemo** [comment]
@lrnv Thanks for you comments!

We have updated the example section now, hope it is easier to understand now. We still try keeping it quite brief, as we were a bit concerned about the length (there tend to be a large number of details for all such case studies).

We discussed the license issue internally, and as we understand it the paper will be licensed under CC By 4.0, while the code is licensed under MIT, and that sounds reasonable to us, but feel free to elaborate if this is something that should be aligned.
